src/main/frontend/src/components/itemPage/BiddersSection.js

```diff
@@ -31,13 +32,14 @@ function BiddersSection({bids}) {
                     <Col>Date</Col>
                     <Col>Bid</Col>
                 </Row>
-                {bidDetails.map((bid) =>
+                {bidDetails.sort((a,b) => b.amount - a.amount).slice(0,limit).map((bid) =>
```

Comment:
Can we sort bidDetails on line 12, where we set bid details, so sorting is done only once and this can be bidDetails.map((bid)...